[2021-09-27 16:34:32,898] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T16:20:00+00:00 [queued]>
[2021-09-27 16:34:32,964] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T16:20:00+00:00 [queued]>
[2021-09-27 16:34:32,968] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 16:34:32,970] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 16:34:32,972] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 16:34:33,012] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): say_hello> on 2021-09-27T16:20:00+00:00
[2021-09-27 16:34:33,029] {standard_task_runner.py:52} INFO - Started process 24311 to run task
[2021-09-27 16:34:33,057] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'say_hello', '2021-09-27T16:20:00+00:00', '--job-id', '79', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpu43_i3g3', '--error-file', '/tmp/tmppa2il2h7']
[2021-09-27 16:34:33,064] {standard_task_runner.py:77} INFO - Job 79: Subtask say_hello
[2021-09-27 16:34:33,266] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.say_hello 2021-09-27T16:20:00+00:00 [running]> on host 7c1d2d2abdb5
[2021-09-27 16:34:33,427] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Veerawat
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=say_hello
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T16:20:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-27T16:20:00+00:00
[2021-09-27 16:34:33,429] {my_dag.py:11} INFO - Hello, SAKSIAM
[2021-09-27 16:34:33,431] {python.py:151} INFO - Done. Returned value was: None
[2021-09-27 16:34:33,463] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=say_hello, execution_date=20210927T162000, start_date=20210927T163432, end_date=20210927T163433
[2021-09-27 16:34:33,527] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 16:34:33,631] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
